data/core/units/goblins/Wolf_Rider.cfg

```diff
@@ -27,9 +27,9 @@
     advances_to=Goblin Knight,Goblin Pillager
     cost=17
     usage=scout
-    description= _ "No one is quite sure how the practice of taming and riding wolves came into being, but the advantages of it for a goblin are obvious. Goblins are smaller and much weaker than their orcish kin, and are often tasked to the most dangerous and unwanted parts of a fight. Any who can manage to win themselves a mount have a much safer, and daresay, more amusing role in combat.
+    description= _ "No one is quite sure how the practice of taming and riding wolves came into being, but the advantages of it for a goblin are obvious. Goblins are smaller and much weaker than their orcish kin, and are often tasked to the most dangerous and unwanted parts of a fight. Any who can manage to win themselves a mount have a much safer role in combat.
 
-Wolves, likely, could never support the weight of a man in plate, but a goblin in leather armor is a simple load to bear. Quite unlike horses, these mounts have a rather easy time traversing the mountains, though water and woods will still slow them down."
+Though even the largest wolves could never support the weight of a man, a goblin is quite within their ability to carry swiftly. Unlike horses, these mounts have a rather easy time traversing the mountains, though water and woods will still slow them down."
```

Comment:
How about "the weight of a true orc" instead of "the weight of a man"?

Reply:
I borrowed the term "true orc" from the race descriptions of orcs and goblins, which both use that term.

Reply:
:+1: 

I'm a bit unsure about the "stand with the weight of a true orc on their back", it sounds somewhat precise/formal. Maybe "could even stand upright bearing the weight of a true orc"? I'm fine with it either way.

Reply:
Thanks, I'll have another go at it; probably "stand up", as "upright" would suggest to me a pose like a horse-show trick, with the horse's front hooves far above the ground.

Low priority anyway, as the 1.14.6 string-freeze has already started.